client/collection_test.go

```diff
@@ -0,0 +1,136 @@
+package client
+
+import (
+	"context"
+	"testing"
+
+	"github.com/davecgh/go-spew/spew"
+	"github.com/stretchr/testify/require"
+	"github.com/tigrisdata/tigris-client-go/config"
+	"github.com/tigrisdata/tigris-client-go/filter"
+	"github.com/tigrisdata/tigris-client-go/projection"
+	"github.com/tigrisdata/tigris-client-go/update"
+)
+
+func TestCollectionBasic(t *testing.T) {
+	ctx := context.TODO()
+
+	client, err := NewClient(ctx, nil)
+	require.NoError(t, err)
+
+	db := client.Database("db1")
+
+	err = db.Create(ctx)
+	require.NoError(t, err)
+
+	type Coll1 struct {
+		Key1   string `tigris:"primary_key""`
+		Field1 int64
+	}
+
+	type Coll2 struct {
+		tigris.Model        // optional, if embedded CreatedAt, ... filled automatically from the response
+		Key1         string `tigris:"primary_key""`
+		Field1       int64
+	}
+
+	err = db.CreateCollections(ctx, "db1", &Coll1{}, &Coll2{}, &Coll3{}, &Coll4{})
```

Comment:
can we avoid having to pass the database name as a parameter?

Reply:
Actually, yet. Fixed.